### PR TITLE
Ohio county road shields

### PR DESCRIPTION
**Pull request by 1ec5**
Added shields for all the counties in Ohio that signpost their county roads with shields, other than the ones that post black text on a white rectangle, which is already the fallback shield.

[Mahoning](https://zelonewolf.github.io/openstreetmap-americana/#12.04/41.11198/-80.82983)<br>_Mahoning County_

[Medina–Wayne](https://zelonewolf.github.io/openstreetmap-americana/#12.12/40.9974/-82.08908)<br>_Medina County (top) and Wayne County (bottom)_

[Vinton–Athens](https://zelonewolf.github.io/openstreetmap-americana/#13.99/39.25122/-82.26493)<br>Vinton County (left) and Athens County (right)_

[Ashland–Wayne](https://zelonewolf.github.io/openstreetmap-americana/#15.12/40.778914/-82.126555)<br>_Ashland County (left) and Wayne County (right)_

[Scioto](https://zelonewolf.github.io/openstreetmap-americana/#16.05/38.692246/-82.856904)<br>_Scioto County_

[Williams](https://zelonewolf.github.io/openstreetmap-americana/#12.33/41.61831/-84.61263)<br>_Williams County_

### CODE REVIEW

File: style/js/shield_defs.js [hunk 1]
```diff
@@ -384,6 +384,48 @@ export function loadShields(shieldImages) {
...
+      shieldImages.shield40_us_oh_asd,
+    ],
+    textColor: "#006747",
+    padding: {
+      left: 6,
+      right: 3,
+      top: 4,
+      bottom: 7,
+    },
+  };
```
Comment (1ec5) [inline style/js/shield_defs.js:414]: Four-digit Ashland County road numbers are practically illegible, but I’ve already distorted the county shape to be barely recognizable. The number is practically illegible in real life anyways.

[Ashland Co. 1095](https://commons.wikimedia.org/wiki/File:Ashland_County_Route_1095_OH.svg)

File: style/js/shield_defs.js [hunk 2]
```diff
@@ -384,6 +384,48 @@ export function loadShields(shieldImages) {
...
+      shieldImages.shield40_us_oh_sci_3,
+    ],
+    textColor: "black",
+    padding: {
+      left: 3,
+      right: 3,
+      top: 4,
+      bottom: 6,
+    },
+  };
```
Comment (1ec5) [inline style/js/shield_defs.js:427]: I put an orange stroke around the Scioto County shape, even though the shield gives it a black stroke on an orange field. A black stroke would give it much better contrast against the background but less contrast against nearby Ohio state route shields.

[Scioto CR 40](https://commons.wikimedia.org/wiki/File:Scioto_County_Route_40_OH.svg)

**Comment by 1ec5** [comment]
For reference:

[Ohio counties and their route shields](https://commons.wikimedia.org/wiki/File:Ohio’s_counties_and_their_county_route_shields.svg)

**Comment by ZeLonewolf** [comment]
My god these are hideous.  Great job accurately capturing the on the job situation in Ohio!

**Comment by zekefarwell** [comment]
I'd say specifically the Ashland and Scioto County blobs are hideous 😂.  The rest are quite sensible rectangles or pentagons that look good!

**Comment by ZeLonewolf** [comment]
The phrase "thanks, I hate it" comes to mind here.

File: style/icons/shield40_us_oh_asd.svg [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="20" height="20" version="1.0" xmlns="http://www.w3.org/2000/svg" xmlns:cc="http://creativecommons.org/ns#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+ <path id="Ashland" d="m14.493 0.5v2.5h3s0.03432 8.345 0 12.36c-0.91575-0.0023-3.0728-0.0045-3.9886-0.0068l-0.022621 2.6468-7.489 1.5c0.015187-2.1256 0-6 0-6h-2v-2.6662l-1.5-1.3768v-6.2979l-1-0.38877 0.00882-2.2703z" fill="#fff" stroke="#006747" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.0001"/>
+ <metadata>
+  <rdf:RDF>
+   <cc:Work rdf:about=""/>
+  </rdf:RDF>
+ </metadata>
```
Comment (ZeLonewolf) [inline style/icons/shield40_us_oh_asd.svg:8]: Looks like there's some extra stuff in the SVGs that we can remove.

File: style/js/shield_defs.js [hunk 4]
```diff
@@ -384,6 +384,81 @@ export function loadShields(shieldImages) {
...
+    "HEN",
+    "MOE",
+    "PAU",
+    "WAS",
+    "WIL",
+    "WYA",
+  ].forEach(
+    // White on green rectangle
+    (county) =>
+      (shields["US:OH:" + county] = roundedRectShield(
```
Comment (ZeLonewolf) [inline style/js/shield_defs.js:405]: Can these use the templated form:
```
[`US:OH:${county}`] ?
 ```

**Review by ZeLonewolf — CHANGES_REQUESTED** [review]
Styling looks good in the map and I concur with your design choices.  Just a couple cleanup/code style notes.

**Review by ZeLonewolf — APPROVED** [review]
Looks good, everything seems to be in order.

**Pull request merged**